nixnet/session.py

```diff
@@ -22,22 +22,76 @@
     "FrameOutStreamSession",
     "FrameInQueuedSession",
     "FrameOutQueuedSession",
+    "FrameInSinglePointSession",
+    "FrameOutSinglePointSession",
     "SignalInSinglePointSession",
     "SignalOutSinglePointSession"]
 
 
 class SessionBase(object):
+    """Session base object.
```

Comment:
I've moved the code to _session/base.py and updated the documentation as needed.